src/components/quickReplyModal/quickReplyModalView.tsx

```diff
@@ -0,0 +1,228 @@
+import get from 'lodash/get';
+import { navigate } from '../../navigation/service';
+
+export interface QuickReplyModalProps {}
+
+const QuickReplyModal = ({}: QuickReplyModalProps, ref) => {
+  const intl = useIntl();
+  const dispatch = useDispatch();
+  const currentAccount = useSelector((state) => state.account.currentAccount);
+  const pinCode = useSelector((state) => state.application.pin);
+  const isLoggedIn = useAppSelector((state) => state.application.isLoggedIn);
+
+  const [selectedPost, setSelectedPost] = useState(null);
+  const [commentValue, setCommentValue] = useState('');
+  const [isSending, setIsSending] = useState(false);
+  const [isKeyboardVisible, setKeyboardVisible] = useState(false);
+  const sheetModalRef = useRef<ActionSheet>();
+  const inputRef = useRef<TextInput>(null);
+
+  useEffect(() => {
+    const keyboardDidShowListener = Keyboard.addListener('keyboardDidShow', () => {
+      setKeyboardVisible(true); // or some other action
+    });
+    const keyboardDidHideListener = Keyboard.addListener('keyboardDidHide', () => {
+      setKeyboardVisible(false); // or some other action
+    });
+
+    return () => {
+      keyboardDidHideListener.remove();
+      keyboardDidShowListener.remove();
+    };
+  }, []);
+
+  // reset the state when post changes
+  useEffect(() => {
+    setCommentValue('');
+  }, [selectedPost]);
+
+  //CALLBACK_METHODS
+  useImperativeHandle(ref, () => ({
+    show: (post: any) => {
+      console.log('Showing action modal');
+      setSelectedPost(post);
+      sheetModalRef.current?.setModalVisible(true);
+      // wait  for modal to open and then show the keyboard
+      setTimeout(() => {
+        inputRef.current?.focus();
+      }, 1000);
+    },
+  }));
+
+  // handlers
+
+  // navigate to post on summary press
+  const _handleOnSummaryPress = () => {
+    Keyboard.dismiss();
+    sheetModalRef.current?.setModalVisible(false);
+    navigate({
+      routeName: ROUTES.SCREENS.POST,
+      params: {
+        content: selectedPost,
+      },
+      key: get(selectedPost, 'permlink'),
+    });
+  };
+
+  // handle submit reply
+  const _submitReply = async () => {
+    let stateTimer;
+    if (isSending) {
+      return;
+    }
+
+    if (currentAccount) {
+      setIsSending(true);
+
+      const permlink = generateReplyPermlink(selectedPost.author);
+
+      const parentAuthor = selectedPost.author;
+      const parentPermlink = selectedPost.permlink;
+      const parentTags = selectedPost.json_metadata.tags;
+      console.log(
+        currentAccount,
+        pinCode,
+        parentAuthor,
+        parentPermlink,
+        permlink,
+        commentValue,
+        parentTags,
+      );
+
+      const status = await postComment(
+        currentAccount,
+        pinCode,
+        parentAuthor,
+        parentPermlink,
+        permlink,
+        commentValue,
+        parentTags,
+      )
+        .then(() => {
+          AsyncStorage.setItem('temp-reply', '');
```

Comment:
We do not need to clear `temp-reply` here as we are not making use of temp-reply in quick reply modal, it is only being used for full comment editor.

Reply:
@aliseyalvi rather what we should do is clear text state, the attached screenshot show same comment after it has been successfully made, it should appear empty here....

<img width="331" alt="Screenshot 2022-01-13 at 10 47 50 PM" src="https://user-images.githubusercontent.com/6298342/149382487-99928b1f-ec73-4e00-aba3-e2cbf9a11d2d.png">